src/main/java/Todo.java

```diff
@@ -0,0 +1,9 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+    @Override
+    public String toString() {
+        return ("[T]" + super.toString() + this.description.split(" ",2)[1]);
+    }
+}
```

Comment:
Task and Todo are functionally identical at this point (apart from returning "[T]"). Could they be merged into a single class?